syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTVerificationSet";
option optimize_for = LITE_RUNTIME;

import public "VerificationGroup.proto";

message VerificationSet {
    optional uint32 version = 1;
    optional VerificationGroup internal = 2;
    optional VerificationGroup external = 3;
    repeated string repudiated = 4;
}
